src/scales/categoryScale.ts

```diff
@@ -89,14 +89,17 @@ export class Category extends Scale<string, number> implements TransformableScal
    * @returns {string[]}
    */
   public invertRange(range: [number, number] = this.range()): string[] {
-    const rangeBand = this._d3Scale.rangeBand();
+    const rangeBand = this._d3Scale.bandwidth();
     // offset the domain by half the rangeBand such that we consider the
     // center of the bars
-    const domainStartNormalized = this.invertedTransformation(range[0]) - rangeBand / 2;
```

Comment:
what about this half rangeBand offset? If it's no longer needed maybe we can remove the comment above